docs/quickstarts/ralph.rst

```diff
@@ -0,0 +1,36 @@
+.. _quick-start-ralph:
+
+Ralph
+******
+
+Ralph is the default option to send xAPI events to Clickhouse. To run it
+make sure to enable the `RUN_RALPH` option in the `config.yml` file.
+
+.. code-block:: yaml
+
+    RUN_RALPH: True
+
+Aspects provides the following configuration options:
+
+.. code-block:: yaml
+    
+    RALPH_ENABLE_PUBLIC_URL: False # If True, the public URL ralph.{{LMS_HOST}} will be used instead of the internal one
```

Comment:
Should note that you will need to make this True if you want Ralph to be used by external services